### PR TITLE
Add event prime links

### PR DESCRIPTION
**Pull request by ooade**

### CODE REVIEW

**Comment by ooade** [comment]
@Developerayo "Save your seat" currently does nothing. What should be the default behaviour?

**Comment by Developerayo** [comment]
"Save your seat" function should also lead to the registration link on eventprime. @ooade

**Comment by ooade** [comment]
The one on the nav goes to another page?

File: about.html [hunk 1]
```diff
@@ -86,7 +86,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline about.html]: target this link outside also

Comment (Developerayo) [inline about.html]: target this link outside also

File: contact.html [hunk 2]
```diff
@@ -86,7 +86,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline contact.html]: Target out

File: index.html [hunk 3]
```diff
@@ -112,7 +112,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline index.html]: target out

File: index.html [hunk 4]
```diff
@@ -155,7 +155,7 @@ <h2>unStack <br>Nigeria</h2>
                                     where learning is done on a whole other level through hands-on workshops and
                                     world-class talks.⚡️</div>
                                 <div class="btn-box clearfixr">
-                                    <a href="#" class="theme-btn btn-style-two">Save your seat</a>
+                                    <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-two">Save your seat</a>
```
Comment (Developerayo) [inline index.html]: target out

File: mission.html [hunk 5]
```diff
@@ -86,7 +86,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline mission.html]: target out

File: organizers.html [hunk 6]
```diff
@@ -86,7 +86,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline organizers.html]: target out

File: past-events.html [hunk 7]
```diff
@@ -89,7 +89,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline past-events.html]: target out

File: schedule.html [hunk 8]
```diff
@@ -86,7 +86,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline schedule.html]: target out

File: speakers.html [hunk 9]
```diff
@@ -86,7 +86,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline speakers.html]: target out

File: sponsors.html [hunk 10]
```diff
@@ -89,7 +89,7 @@
                             <!-- btn box -->
                             <div class="btn-box">
                                 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
-                                <a href="contact.html" class="theme-btn btn-style-one">Save a seat</a>
+                                <a href="https://eventprime.co/e/react-present-and-future-a-hands-on-approach" target="_blank" class="theme-btn btn-style-one">Save a seat</a>
```
Comment (Developerayo) [inline sponsors.html]: target out

**Comment by ooade** [comment]
What's target out? 🌚

**Comment by Developerayo** [comment]
> @ooade Target the links to a new tab `target="_blank"`
> As for the Save a Seat, add the same link to it and target it to a new tab.

target the save a seat button to a new tab as you did for register @ooade

**Comment by ooade** [comment]
They all have target _blank 🙄

**Comment by Developerayo** [comment]
Haha, my eyes skipped it, hehehe

**Comment by Developerayo** [comment]
Great job Man!

**Pull request merged**